contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -280,112 +180,107 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, Reentrancy
         }
     }
 
-    /// @notice Allows to claim revenue tokens on behalf of the ip royalty vault by snapshot batch
-    /// @param snapshotIds The list of snapshot ids
+    /// @notice Get total amount of revenue token claimable by a token holder
+    /// @param claimer The address of the token holder
```

Comment:
The address of the token holder

could be

The address of the royalty token holder

Reply:
changed.